### PR TITLE
 Improve admin

### PR DESCRIPTION
**Pull request by phamhuuluan**
==> Tittle: Task Task #22503 Improve admin

==> Redmine: https://edu-redmine.sun-asterisk.vn/issues/22503

==> Attachment [File:](url
Screenshot from 2020-03-25 13-31-22
)

[
Screenshot from 2020-03-25 13-32-03
](url)

### CODE REVIEW

**Comment by phamhuuluan** [comment]
ready

File: app/views/admins/users/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      <%= f.text_field :fullname, class: "form-control" %>
+
+      <%= f.label :email, t(".email") %>
+      <%= f.text_field :email, class: "form-control" %>
+
+      <%= f.label :password, t(".password") %>
+      <%= f.password_field :password, autofocus: true, autocomplete: "current-password", class: "form-control",
+        placeholder: t(".password"), required: true %><br>
+
+      <div class="form-group ">
```
Comment (ngocvt-0484) [inline app/views/admins/users/_form.html.erb]:
```suggestion
      <div class="form-group">
```

File: app/views/admins/users/new.html.erb [hunk 2]
```diff
@@ -0,0 +1 @@
+<%= render partial: "admins/users/form" %>
```
Comment (ngocvt-0484) [inline app/views/admins/users/new.html.erb]: render "form" là dc r mà e

File: app/views/admins/users/edit.html.erb [hunk 3]
```diff
@@ -0,0 +1 @@
+<%= render partial: "admins/users/form" %>
```
Comment (ngocvt-0484) [inline app/views/admins/users/edit.html.erb]: render "form" là dc r mà e

File: app/views/admins/users/_form.html.erb (hunk 1 above)
Comment (phamhuuluan) [reply app/views/admins/users/_form.html.erb]: dạ vâng ạ

File: app/views/admins/users/new.html.erb (hunk 2 above)
Comment (phamhuuluan) [reply app/views/admins/users/new.html.erb]: dạ vâng chị

File: app/views/admins/users/edit.html.erb (hunk 3 above)
Comment (phamhuuluan) [reply app/views/admins/users/edit.html.erb]: dạ

**Comment by phamhuuluan** [comment]
ready

**Pull request merged**